client/core/core.go

```diff
@@ -1126,9 +1125,9 @@ func New(cfg *Config) (*Core, error) {
 			}
 			langs = append(langs, tag)
 		}
-		// fmt.Println("defined langs:", langs)
 		matcher := language.NewMatcher(langs)
-		tag, _, conf := matcher.Match(acceptLangs...)
+		_, idx, conf := matcher.Match(acceptLang) // use index because tag may end up as something hyper specific like zh-Hans-u-rg-cnzzzz
+		tag := langs[idx]
```

Comment:
We needed to be using the index from the matcher, like we were doing in webserver.go for the reason described in the comment.